nss-day-cohort-52/nashville-kennels-cocoa-foxes
# Employee Fire Button

Description:

Fire button on each individual employee card is only rendered if the currently logged-in user is an employee. Clicking on the fire button for an employee will immediately remove them from the API and the updated employee list will be rendered to the user.


Related Issue Tickets:

Fixes #2 
Fixes #10 


Steps to Test:
1. Provided that the currently logged-in user is an employee, navigate to the employees list using the nav bar. The user may then click on an employee name to view their individual card and a "fire" button will appear at the bottom of the card. Clicking on the "fire" button should result in a 200 DELETE Network call in the dev tools for the associated userId and the page will immediately refresh to the updated employee list.
2. Provided that the currently logged-in user is NOT an employee, navigate to the employees list using the nav bar. The user may then click on an employee name to view their individual card and the "fire" button will NOT appear at the bottom.
